CRM/Contribute/Form/ContributionBase.php

```diff
@@ -1425,4 +1425,25 @@ protected function initializeOrder(): void {
     $this->order->setForm($this);
   }
 
+  /**
+   * Get the currency for the form.
+   *
+   * Rather historic - might have unneeded stuff
+   *
+   * @return string
+   */
+  public function getCurrency() {
+    $currency = $this->_values['currency'] ?? NULL;
+    // For event forms, currency is in a different spot
```

Comment:
The unneeded stuff comment likely correct - probably safe to remove that in this PR?